test/attribute_cache_test.rb

```diff
@@ -43,7 +43,7 @@ def test_cached_attribute_values_are_expired_from_the_cache_when_an_existing_rec
     assert_queries(1) { assert_equal 'foo', AssociatedRecord.fetch_name_by_id(1) }
     assert_queries(0) { assert_equal 'foo', AssociatedRecord.fetch_name_by_id(1) }
 
-    @record.save!
+    @record.update!(updated_at: @record.updated_at + 1)
```

Comment:
Would a `touch` be easier to understand here/cleaner?

Reply:
I wanted to avoid having the test fail because the record's `updated_at` matched the current time, in which case we are trying to skip expiring the cache.

Reply:
ah right, the timestamp isn't precise enough, makes sense